packages/data-context/src/data/coreDataShape.ts

```diff
@@ -71,6 +71,7 @@ export interface WizardDataShape {
   detectedBundler: WizardBundler | null
   detectedFramework: Cypress.ResolvedComponentFrameworkDefinition | null
   frameworks: Cypress.ResolvedComponentFrameworkDefinition[]
+  erroredFrameworks: Cypress.ErroredFramework[]
```

Comment:
Since erroredFrameworks isn't part of the public API, I don't think we want to expose that type on `Cypress`